src/runtime/audience-action-flow.js

```diff
@@ -186,16 +186,22 @@ export class AudienceActionFlow {
   showSignedInToast_(userEmail) {
     const lang = this.clientConfigManager_.getLanguage();
     let customText = '';
-    if (this.params_.action === 'TYPE_REGISTRATION_WALL') {
-      customText = msg(
-        SWG_I18N_STRINGS.REGWALL_ACCOUNT_CREATED_LANG_MAP,
-        lang
-      ).replace(placeholderPatternForEmail, userEmail);
-    } else if (this.params_.action === 'TYPE_NEWSLETTER_SIGNUP') {
-      customText = msg(
-        SWG_I18N_STRINGS.NEWSLETTER_SIGNED_UP_LANG_MAP,
-        lang
-      ).replace(placeholderPatternForEmail, userEmail);
+    switch (this.params_.action) {
+      case 'TYPE_REGISTRATION_WALL':
+        customText = msg(
+          SWG_I18N_STRINGS.REGWALL_ACCOUNT_CREATED_LANG_MAP,
+          lang
+        ).replace(placeholderPatternForEmail, userEmail);
+        break;
+      case 'TYPE_NEWSLETTER_SIGNUP':
+        customText = msg(
+          SWG_I18N_STRINGS.NEWSLETTER_SIGNED_UP_LANG_MAP,
+          lang
+        ).replace(placeholderPatternForEmail, userEmail);
+        break;
+      default:
+        // Do not show toast for other types.
+        return;
```

Comment:
Would it be possible to add test coverage for this line?

Reply:
Added!

Reply:
Thanks!